doc/modules/clustering.rst

```diff
@@ -1582,3 +1582,35 @@ Drawbacks
  *  Caliński, T., & Harabasz, J. (1974). "A dendrite method for cluster
     analysis". Communications in Statistics-theory and Methods 3: 1-27.
     `doi:10.1080/03610926.2011.560741 <http://dx.doi.org/10.1080/03610926.2011.560741>`_.
+
+.. _contingency_matrix:
+
+Contingency Matrix
+----------------------
+
+Contingency matrix is a format describing the relationship between
+discrete-valued random variables.
+It shows the multivariate frequency distribution of the variables.
```

Comment:
I don't think this technical language is helpful. And it is imprecise: the matrix doesn't, for instance, give a probability distribution.

I also don't think we elsewhere describe cluster predictions as random variables.

The important thing about a contingency matrix in clustering is that that the order of columns and rows is arbitrary. Your definition of random variables misses this too.

Reply:
Better off describing it in terms of a clustering as a set of sets: contingency matrix reports the intersection cardinality for every true/predicted cluster pair.

Reply:
The contingency matrix provides sufficient statistics for all clustering metrics where the samples are iid and you don't need to account for some instances not being clustered.

Reply:
> the matrix doesn't, for instance, give a probability distribution.

That line doesn't say this. It says it gives 'frequency distribution' not probability distribution which I think is different from 'frequency distribution'.

> I don't think this technical language is helpful.

I do agree on this point.